v2/internal/gomod/gomod.go

```diff
@@ -112,3 +112,13 @@ func SyncGoVersion(goModText []byte, goVersion string) ([]byte, bool, error) {
 
 	return goModText, true, nil
 }
+
+func UpdateGoModuleName(goModText []byte, newModuleName string) ([]byte, error) {
+	file, err := modfile.Parse("", goModText, nil)
```

Comment:
No doubt this works just fine, but I'm wondering if `go mod edit -module <name>` would be more forward compatible?

Reply:
You're right! This is indeed a lack of consideration on my part. Compared with calling the `golang.org/x/mod/modfile` library to parse `go.mod`, the latter is more forward-compatible using commands. I will update its implementation and update the PR.